docs/conf.py

```diff
@@ -29,10 +29,14 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "sphinx.ext.autodoc",
     "myst_parser",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
 ]
 
+intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
```

Comment:
Automatically links our documentation to the documentation of any Python standard library member we reference, either as subclass or in a type annotation.